pbsmetrics/go_metrics.go

```diff
@@ -562,6 +582,11 @@ func (me *Metrics) RecordTimeoutNotice(success bool) {
 	return
 }
 
+func (me *Metrics) RecordTCFReq(version TCFVersionValue) {
+	me.TCFReqVersion[version].Mark(1)
```

Comment:
If version isn't found in the map, would it be better to hardcode the unknown value or ignore the request instead of potentially getting a nil ref panic?